code/obj/machinery/status_display.dm

```diff
@@ -86,6 +97,10 @@ TYPEINFO(/obj/machinery/status_display)
 		if(!src.net_id)
 			src.net_id = generate_net_id(src)
 
+	disposing()
+		..()
+		src.proxy?.dispose()
```

Comment:
not quite
```suggestion
	disposing()
		qdel(src.proxy)
		src.proxy = null
		. = ..()
		
```

Reply:
why `qdel` instead of `.dispose()` for this use case? because we're in `disposing()`?

Reply:
pretty sure our convention is to never call dispose directly, @pali6?

Reply:
pali said yes